commands/earthevent.js

```diff
@@ -0,0 +1,95 @@
+const { SlashCommandBuilder } = require("discord.js")
+const axios = require("axios").default
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("earthevent")
+    .setDescription(
+      "Return a natural event that has occurred in the last 30 days."
+    ),
+  async execute(interaction) {
+    await axios({
+      method: "get",
+      url: "https://eonet.gsfc.nasa.gov/api/v3/events?limit=60&days=30",
+      responseType: "json",
+    })
+      .then((response) => {
+        // get number of events in the past 30 days, limited to 60
```

Comment:
Sometimes the api doesn't respond in 3 seconds and the reply returns an error.
To fix this, add `await interaction.deferReply({ ephemeral: false })` between line 16 and 17.
But the other `interaction.reply()` functions need to be changed to `interaction.editReply()`.

For reference see crypto.js, series.js or showchessleaderboard.js :)